src/dbos-runtime/config.ts

```diff
@@ -171,13 +190,14 @@ export interface ParseOptions {
   configfile?: string;
   appDir?: string;
   appVersion?: string | boolean;
+  logWhileParsing?: boolean;
 }
 
 /*
  * Parse `dbosConfigFilePath` and return DBOSConfig and DBOSRuntimeConfig
  * Considers DBOSCLIStartOptions if provided, which takes precedence over config file
  * */
-export function parseConfigFile(cliOptions?: ParseOptions, useProxy: boolean = false): [DBOSConfig, DBOSRuntimeConfig] {
+export function parseConfigFile(cliOptions?: ParseOptions): [DBOSConfig, DBOSRuntimeConfig] {
```

Comment:
Not sure about the proxy thing -- @qianl15 or @devhawk are better reviewers for this

Reply:
I think this was a workaround for bypassing the password check. This PR changes the way we handle passwords so we can remove it.